### PR TITLE
Sumo backports v2019.07

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: meta-mender-qemu/scripts/docker/entrypoint.sh [hunk 1]
```diff
@@ -27,11 +27,16 @@ if [ -f /mnt/config/artifact-verify-key.pem ]; then
...
 if [ ! -e /mender-setup-complete ]; then
     ./setup-mender-configuration.py --img="$DISK_IMG" \
                                     --server-url=$SERVER_URL \
-                                    --tenant-token=$TENANT_TOKEN $CONFIG_ARGS
+                                    --tenant-token=$TENANT_TOKEN $CONFIG_ARGS \
+                                    --docker-ip="$DOCKER_IP"
     touch /mender-setup-complete
 fi
 
+export QEMU_NET_HOSTFWD=",hostfwd=tcp::80-:80"
```
Comment (kacf) [inline meta-mender-qemu/scripts/docker/entrypoint.sh]: Don't you also need 021ed34f5d049fcd4f8e4f43b9c620c9a53182e0?

Comment (lluiscampos) [reply meta-mender-qemu/scripts/docker/entrypoint.sh]: Yes, good catch. I reviewed only thud.

Can we wait until thud is updated and then we backport at once? This is anyway for a future feature, and it desn't harm to be incomplete IMO.

Comment (lluiscampos) [reply meta-mender-qemu/scripts/docker/entrypoint.sh]: We can also port the commits from #776 and then re-build and re-test

Comment (kacf) [reply meta-mender-qemu/scripts/docker/entrypoint.sh]: Ah of course. So it's actually my fault! ;-D

Grabbing the commits from #776 sounds good.

Comment (lluiscampos) [reply meta-mender-qemu/scripts/docker/entrypoint.sh]: :+1:

**Comment by lluiscampos** [comment]
New builds after latest cherry-picks:
* 1.7.1 in [Jenkins 3956](https://mender-jenkins.mender.io/job/mender-builder/3956/)
* 2.0.1 in [Jenkins 3957](https://mender-jenkins.mender.io/job/mender-builder/3957/)

**Comment by lluiscampos** [comment]
I updated the back-port with latest changes released in `thud-v2019.08` except for:
* Allow default bootloader to be overridden  8d81c212a4d61d3b9bb5ddc8ee7dff955700d710 (and related commits)
* mender: Setup Live installer with HDDIMG 698ef94741289131ff18cea2d71bda412c7a4e17

Builds in progress:
* 1.7.1 https://mender-jenkins.mender.io/job/mender-builder/4310/
* 2.0.1 https://mender-jenkins.mender.io/job/mender-builder/4311/
* 2.1.0b1 https://mender-jenkins.mender.io/job/mender-builder/4312/

**Comment by lluiscampos** [comment]
@kacf wrt the build failures for 2.1.0b1

It seems like with 2.1.0b1 we are using golang's v1.10 `strings.Builder`. sumo and rocko branches will not be able to cope with it. I assume we will port 2.1.0 final to these branches, so what should we do? Can we just disable the unit tests?

**Comment by kacf** [comment]
You can reuse this: https://github.com/mendersoftware/meta-mender/blob/sumo/meta-mender-core/recipes-mender/mender/mender_2.0.0.bb#L30-L35

**Comment by lluiscampos** [comment]
> You can reuse this: https://github.com/mendersoftware/meta-mender/blob/sumo/meta-mender-core/recipes-mender/mender/mender_2.0.0.bb#L30-L35

Ah, of course! I forgot we already had it for 2.0.x...

**Comment by lluiscampos** [comment]
@kacf All builds for the three versions are green.

**Pull request merged**